config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -465,10 +465,10 @@ config SHELL_STACK_SIZE
     default 2616 if CHIP_WIFI
 
 config SHELL_MINIMAL
-    default y
+    default y if !CHIP_MEMORY_PROFILING
 
 config KERNEL_SHELL
-    default n
+    default n if !CHIP_MEMORY_PROFILING
```

Comment:
Do we need it? Could we just enable KERNEL_SHELL inside CHIP_MEMORY_PROFILING Kconfig definition?

Reply:
I did it to ensure that the order of kconfig includes doesn't matter as Damian noticed above.

Reply:
So indeed, it is not needed then 😄 I thought that it also will be overwritten.